src/assessments/seo/outboundLinksAssessment.js

```diff
@@ -35,16 +53,19 @@ class OutboundLinksAssessment extends Assessment {
 	 *
 	 * @param {Paper} paper The paper to use for the assessment.
 	 * @param {Researcher} researcher The researcher used for calling research.
-	 * @param {object} i18n The object used for translations
+	 * @param {Object} i18n The object used for translations.
 	 *
 	 * @returns {AssessmentResult} The assessment result.
 	 */
 	getResult( paper, researcher, i18n ) {
-		let linkStatistics = researcher.getResearch( "getLinkStatistics" );
+		this.linkStatistics = researcher.getResearch( "getLinkStatistics" );
 		let assessmentResult = new AssessmentResult();
-		if ( ! isEmpty( linkStatistics ) ) {
-			assessmentResult.setScore( this.calculateScore( linkStatistics ) );
-			assessmentResult.setText( this.translateScore( linkStatistics, i18n ) );
+
+		if ( ! isEmpty( this.linkStatistics ) ) {
```

Comment:
I'd consider inverting this if-statement for an early return.

Reply:
Actually, linkStatistics is never Empty, so I just removed this if statement.